src/tests/api/BavUnhappyPath.test.ts

```diff
@@ -103,7 +105,31 @@ describe("BAV CRI unhappy path tests", () => {
 		});
 	});
 
-	describe("/authorization Endpoint Unhappy Path Tests", () => {
+	describe("BAV CRI: /verify-account Endpoint Partial Match Athena Output Test", () => {
+		it("Triggers Partial Match and checks for object in Athena", async () => {
+			// ARRANGE
+			const firstName = randomUUID().slice(-8);
+			const newStubPayload = structuredClone(exampleStubPayload);
+			newStubPayload.shared_claims.name[0].nameParts[0].value = firstName;
+
+			const sessionId = await startStubServiceAndReturnSessionId(newStubPayload);
+
+			const newVerifyAccountYesPayload = structuredClone(verifyAccountYesPayload);
+			newVerifyAccountYesPayload.account_number = "00111114";
+			const startTime = absoluteTimeNow();
+
+			// ACT
+			await sleep(500);
+			const verifyAccountResponse = await verifyAccountPost(newVerifyAccountYesPayload, sessionId);
+
+			// ASSERT
+			expect(verifyAccountResponse.status).toBe(200);
+			const athenaRecords = await getAthenaRecordByFirstNameAndTime(startTime, firstName);
+			expect(athenaRecords.length).toBeGreaterThan(0);
+		});
+	});
```

Comment:
Hmmmm... this will be very hard to do as the object put in to S3 contains a generated UUID in the key (filename), and the key is not externalised. So the test harness will not know what to delete.

However both the name match bucket and the athena results buckets have lifecycle rules in all envs which permanently delete objects after 30 days, so they do eventually get cleaned up.